test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -20,44 +21,90 @@ public SqlOutputBindingIntegrationTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        private async Task<HttpResponseMessage> SendOutputRequest(string functionName, IDictionary<string, string> query = null)
+        private async Task<HttpResponseMessage> SendOutputRequest(string functionName, IDictionary<string, string> query = null, bool asPost = false)
```

Comment:
I'd rather we just split them up into separate functions, they're small enough and do different enough things that that'll be a bit cleaner.